src/components/AddItemForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { withFirestore } from 'react-firestore';
+import '../CSS/AddItemForm.css';
+class AddItemForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      next_purchase: null,
+      last_purchase: null,
+      user_token: 'shpongle',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSchedule = this.handleSchedule.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleSchedule(event) {
+    this.setState({ next_purchase: event.target.value });
+  }
+
+  handleSubmit(event) {
+    this.props.firestore
```

Comment:
Not at all a requirement for this ticket—just a question: How might you redirect the user back to their list after they successfully add an item?